src/encoded/schemas/changelogs/target.md

```diff
@@ -1,11 +1,16 @@
 ## Changelog for target.json
 
-### Schema version 8
+### Schema version 9
 
-* Move status property to standard_status mixin.
 * Added "genes" property for the list of genes targeted by assay or antibody associated with the target.
+* Removed the gene_name property.
 * The "organism" property is now a calculated property. The organism of a target is either specified directly through the new "target_organism" property or defined indirectly by its target genes. The "target_organism" property and the "genes" property are mutually exclusive.
 * Added "modification" property to represent modification(s) made to the corresponding wild-type gene product.
+* Added "synthetic tag" and dependency to address synthetic targets (FLAG) properly.
+
+### Schema version 8
```

Comment:
From the diff it looks like some of these properties were added to version 8 at some point. FYI you can add new changes that didn't require upgrade using `Minor changes` header to keep them separate from the changes that did require a version bump, e.g. https://github.com/ENCODE-DCC/encoded/blob/2445f38faeb454fc12fa5091c85139cce928c613/src/encoded/schemas/changelogs/analysis_step_version.md#minor-changes-since-schema-version-4

Reply:
It may not make sense here but the previous change (ENCD-3998) should have a version bump (we do have `target_8_9`) but I forgot to update the changelog.